src/main/archetype/ui.tests/test-module/lib/wdio-pixelmatch-service/comparisonstrategy.js

```diff
@@ -0,0 +1,21 @@
+const logger = require('@wdio/logger').default;
```

Comment:
Can we place the png images inside a folder name which is meaningful ? For example, inside a top level namespace  `forms/themes`

Reply:
@suryansh, you can move images as mentioned in https://github.com/adobe/aem-project-archetype/pull/802#issuecomment-919670141

Reply:
done